lib/location/city.ex

```diff
@@ -1,17 +1,23 @@
 defmodule Location.City do
   @ets_table :geonames
+  @ets_table_by_label :geonames_by_label
 
   defstruct [:id, :name, :country_code]
 
   def load(heir) do
+    :ets.new(@ets_table, [:named_table, :compressed, {:heir, heir, []}])
+    :ets.new(@ets_table_by_label, [:named_table, :compressed, {:heir, heir, []}])
+
     tab = :binary.compile_pattern("\t")
-    ets = :ets.new(@ets_table, [:named_table, :compressed, {:heir, heir, []}])
 
     File.stream!(source_file())
     |> Stream.map(&String.split(&1, tab))
     |> Enum.each(fn [id, name, country_code] ->
       id = String.to_integer(id)
-      :ets.insert(ets, {id, {name, String.trim(country_code)}})
+      country_code = String.trim(country_code)
+
+      :ets.insert(@ets_table, {id, {name, country_code}})
+      :ets.insert(@ets_table_by_label, {{name, country_code}, id})
```

Comment:
👋 @vinibrsl 

Would duplicate `{name, country}` keys be a problem? For example, there can be multiple cities in a country with the same name, like [Springield](https://en.wikipedia.org/wiki/Springfield) If the `ets` mode is `set`, I guess only the last one inserted would be used for the lookups.

Reply:
Hey! I had to recreate this same PR (new link: https://github.com/plausible/location/pull/3). I'll address your question there. :)